README.md

```diff
@@ -183,6 +185,28 @@ try {
 }
 ```
 
+### Registering Payment or Login without evaluating its risk assessment
+
+Turning off the risk assessment evaluation allows you to register a new transaction (Login or Payment), but the response (`TransactionAssessment`) will be empty. For instance, if you're using the risk assessment only for some payment transactions, you should still register all the other ones: this will avoid any bias on the risk assessment computation.
+
+To register a login or a payment without evaluating its risk assessment, you should use the `evaluateTransaction` boolean set to false
+
+Example:
+
+
+```java
+RegisterLoginRequest registerLoginRequest =
+        RegisterLoginRequest.builder()
+          .installationId("installation id")
+          .accountId("account id")
+          .externalId("external id")
+          .evaluateTransaction(false)
+          .build();
+```
+Would return an empty risk assessment response:
+
+``{}``
```

Comment:
I believe this could be removed, as line 190 already mentions that the response will be empty
```suggestion

```